float64[] point_a
float64[] point_b
---
float64[] data
